.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: elixir
 elixir:
-  - 1.2
```

Comment:
Do we really want to drop the test case for Elixir 1.2 here?  We're still testing Sugar on that version.